nsist/__init__.py

```diff
@@ -282,7 +283,7 @@ def prepare_shortcuts(self):
         copy to the build directory. Prepare target and parameters for these
         shortcuts.
 
-        Also copies shortcut icons
+        Also copies shortcut icons, and, if specified, the license file.
```

Comment:
It doesn't make sense to me to add this to `prepare_shortcuts()`, but I don't think any of the other methods are right either. Let's add a new `copy_license()` method for this.